app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
+  has_many :likes
+  has_many :comments
+
+  def update_likes_counter_for_post
+    update(LikesCounter: likes.count)
+  end
```

Comment:
- [ ] This method for updating the like counter for a post should be implemented in the Like model kindly fix this

Reply:
Updated